apps/project/schema.py

```diff
@@ -548,6 +548,30 @@ class Meta:
         filterset_class = PublicProjectByRegionGqlFileterSet
 
 
+class ProjectActivityList(graphene.ObjectType):
+    count = graphene.Int()
+    project = graphene.Field(ProjectType)
+
+
+class EntryActivityList(graphene.ObjectType):
+    project = graphene.Field(ProjectType)
+    count = graphene.Int()
+    date = graphene.Date()
+
```

Comment:
Type names are global, let's not use a generic name to avoid conflicts.

Also, let's not add an object like project for nested/listing types.

```suggestion
class ProjectSummaryStatProjectActivityType(graphene.ObjectType):
    project_id = graphene.ID()
    count = graphene.Int()


class ProjectSummaryStatEntryActivityType(graphene.ObjectType):
    project_id = graphene.ID()
    count = graphene.Int()
    date = graphene.Date()
```

Reply:
@thenav56 How about we create a generic type for this kind of stats?